addons/pos_restaurant/static/tests/tours/pos_restaurant.js

```diff
@@ -237,3 +238,25 @@ registry.category("web_tour.tours").add("BillScreenTour", {
         BillScreen.isQRCodeShown(),
     ].flat(),
 });
+
+registry.category("web_tour.tours").add("ComboPreparationReceiptTour", {
+    steps: () =>
+        [
+            ProductScreen.confirmOpeningPopup(),
+            FloorScreen.clickTable("5"),
+            ProductScreen.clickHomeCategory(),
+            ProductScreen.clickDisplayedProduct("Office Combo"),
+            combo.select("Combo Product 2"),
+            combo.select("Combo Product 4"),
+            combo.select("Combo Product 6"),
+            combo.confirm(),
+            ProductScreen.clickOrderButton(),
```

Comment:
You should ensure that the button is highlighted before clicking:

```suggestion
            {
                trigger: ".actionpad .submit-order.highlight.btn-primary",
            },
```

You may also modify the `clickOrderButton`'s trigger to include the missing classes.